tests/queries/0_stateless/01453_fixsed_string_sort.sql

```diff
@@ -0,0 +1,21 @@
+drop table if exists badFixedStringSort;
+CREATE TABLE IF NOT EXISTS badFixedStringSort (uuid5_old FixedString(16), subitem String) engine=MergeTree order by  tuple();
+
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '1');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '2');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '1');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '2');
+
+INSERT INTO badFixedStringSort values (UUIDStringToNum('8ad8fc5e-a49e-544c-98e6-1140afd79f80'), '2');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('8ad8fc5e-a49e-544c-98e6-1140afd79f80'), '1');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('8ad8fc5e-a49e-544c-98e6-1140afd79f80'), '2');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('8ad8fc5e-a49e-544c-98e6-1140afd79f80'), '1');
+
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '1');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '2');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '1');
+INSERT INTO badFixedStringSort values (UUIDStringToNum('999e1140-66ef-5610-9c3a-b3fb33e0fda9'), '2');
+
```

Comment:
Maybe add this lines either?
```sql
SELECT cityHash64(groupArray(subitem)) FROM (select * from badFixedStringSort ORDER BY  uuid5_old ASC, subitem ASC);
SELECT cityHash64(groupArray(subitem)) FROM (select * from badFixedStringSort ORDER BY  uuid5_old ASC, subitem ASC settings max_block_size=1);

system start merges badFixedStringSort;
optimize table badFixedStringSort final;

SELECT cityHash64(groupArray(subitem)) FROM (select * from badFixedStringSort ORDER BY  uuid5_old ASC, subitem ASC);
SELECT cityHash64(groupArray(subitem)) FROM (select * from badFixedStringSort ORDER BY  uuid5_old ASC, subitem ASC settings max_block_size=1);
```

Reply:
Ok, I'll add `optimize table` because the bug appears only after merge.